### PR TITLE
Update help mode with new editor command.

### PR DESCRIPTION
**Pull request by svi-n**
#### What does this PR do?
Follow up to https://github.com/monome/teletype/pull/259/commits with help_mode.c changes.

Update live mode dashboard shortcut to shift-d.

#### I have,
* [n/a] updated `CHANGELOG.md` and `whats_new.md`
* [x] updated the documentation
* [x] updated `help_mode.c` (if applicable)
* [x] run `make format` on each commit
* [x] run tests

### CODE REVIEW

File: module/help_mode.c [hunk 1]
```diff
@@ -66,6 +66,8 @@ const char* help1[HELP1_LENGTH] = { "1/16 HELP",
                                     "ALT-SLASH|DISABLE LINE(S)",
                                     "ALT-DELETE|DELETE LINE(S)",
                                     "ALT-UP/DN|MOVE LINE(S)",
+				    "ALT-D|DELETE WORD FORWARD",
```
Comment (scanner-darkly) [inline module/help_mode.c:69]: the indentation here seems off?

**Comment by svi-n** [comment]
I did notice that the live mode dashboard alt-d command overshadows the edit mode alt-d command though. Do we want to use shift-d for dashboard?

**Comment by scanner-darkly** [comment]
i guess we have to. grid visualizer is alt-g, so alt-d is more consistent but consistent editing shortcuts are more important. are you comfortable making the change?

**Comment by svi-n** [comment]
Good to go

**Review by scanner-darkly — APPROVED** [review]

**Comment by desolationjones** [comment]
What does alt-d do in edit mode? I'm seeing nothing in the manual.
Introducing shift-[letter] shortcuts to live or edit mode is going to trip up fools like me who habitually hold the shift key when typing the first letter of the line... Maybe this will help break me of the habit?

**Comment by scanner-darkly** [comment]
@desolationjones it turns on the dashboard view, you need to enter something in the scene description, otherwise there is nothing to display!

**Pull request merged**